tests/testthat/test-utils_gcp.R

```diff
@@ -0,0 +1,199 @@
+tar_test("gcp_gcs_exists()", {
```

Comment:
Could we remove this file and rely on `tests/gcp/test-utils_gcp.R` instead? I would prefer these tests to not be fully automated because that risks running up a lot of expenses and potentially accumulating dangling resources if cleanup steps are not successful.

Reply:
Sure, how do you run these tests then?

Reply:
As painful and clunky as it sounds, I run them manually/interactively when I am about to release a new version.